### ISSUE TITLE
[ui] "Create a dashboard" form is messy

### ISSUE THREAD

**Issue by vvv** [zinclabs/zincobserve#502]
### Which ZincObserve functionalities are the source of the bug?

dashboards

### Description

![Screenshot of the "Create a dashboard” form](https://user-images.githubusercontent.com/17462/229902055-9208bd95-d13c-4bca-a5ea-c299740cd20e.png)

1. `Create a dashboard` would be a better title, IMO. We can't create an _old_ dashboard, can we? Also there is no need to capitalise the word `dashboard`, as it's not the first word of the sentence.
2. A superscript asterisk is perceived as a [“conditions apply”](https://legalvision.com.au/advertising-and-marketing-series-disclaimers-and-fine-print/) footnote. But that's not what it was supposed to mean, is it? If the intention is to denote the required field then we should either a) use different colour (red) for the asterisk, or b) replace the asterisk with ` <em>(required)</em>`, or c) drop the asterisk and mark the _optional_ field instead (see the screenshot of a Discourse sign-up form below). I vote for option (c). 
3. Those `of Dashboard` snippets are noise and should be removed. Even if the user forgot which button they clicked a second ago, the title of the form provides enough context.
4. `Enter Type Description of New Dashboard` is nonsense. I'd suggest to shorten the label to `Description` and put something meaningful as a hint (e.g. `Describe the dashboard if you like`).

#### Exhibit A: Discourse sign-up form

Note that optional field is clearly marked with ` (optional)`.

![image](https://user-images.githubusercontent.com/17462/229911255-48616bfa-bbe8-4f18-bd3d-3fe340fc8241.png)

### Version

```
v0.3.2-9-g10a024a
```

**Comment by vvv** [comment]
cc @bjp232004 @ktx-kirtan

**Comment by ktx-kirtan** [comment]
@vvv 

1. We have updated it to match with other screens, `Add Dashboard`. Let's discuss if we want to change this to `Create dashboard` or anything else.
2. We have a similar `Create User` sidebar, where we have used `*`. So to match with the same, we have used `*` in the name.
3. I totally agree. Those are removed.
4. Labels are enough, in my opinion. Also on `Create User`, we do not have hints as the labels are sufficient to tell what it means. So we removed it. Let's discuss if we need to add them back.

The above changes are part of PR #553
